apis/v1alpha1/backendpolicy_types.go

```diff
@@ -55,33 +64,43 @@ type BackendPolicySpec struct {
 	// the oldest BackendPolicy.
 	//
 	// Support: Core
+	//
+	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=16
 	BackendRefs []BackendRef `json:"backendRefs"`
 
 	// TLS is the TLS configuration for these backends.
 	//
 	// Support: Extended
+	//
 	// +optional
-	TLS *BackendTLSConfig `json:"tls,omitempty"`
+	TLS BackendTLSConfig `json:"tls,omitempty"`
 }
 
 // BackendRef identifies an API object within a known namespace that defaults
 // group to core and resource to services if unspecified.
```

Comment:
This comment does not agree with the spec below. I think I'd previously assumed this meant these values would be inferred by a controller, but now that we're formally marking both Group and Kind required, I think we should remove this as well.

Reply:
Agreed.